src/machine learning analysis.py

```diff
@@ -0,0 +1,170 @@
+# author: Siqi Tao
+# date: 2021-11-26
+
+"""Downloads data as a csv file from online to a local filepath via a URL.
+Usage: download_script.py --url=<url> --filepath=<filepath> 
+ 
+Options:
+--input=<input>       The path or filename pointing to the data
+--prefix=<prefix>     The prefix where to write the output figure(s)/table(s) to and what to call it 
+"""
+
+import os
+import pandas as pd
+from docopt import docopt
+import string
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from sklearn.compose import ColumnTransformer, make_column_transformer
+from sklearn.dummy import DummyClassifier, DummyRegressor
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_selection import RFE, RFECV
```

Comment:
Don't need becauase we have separate feature selections script

Reply:
Fixed